packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/__tests__/quanticGeneratedAnswer.test.js

```diff
@@ -724,15 +742,50 @@ describe('c-quantic-generated-answer', () => {
         generatedAnswerState = initialGeneratedAnswerState;
       });
 
-      it('should not display the generated answer card', async () => {
-        const element = createTestComponent();
-        await flushPromises();
+      describe('when no custom error message slot is provided', () => {
```

Comment:
in this test `cannotAnswer ` is not set to true so this test will pass regardless of if a slot is provided or not.